SpokestackTests/TFLiteWakewordRecognizerTest.swift

```diff
@@ -14,11 +14,11 @@ import XCTest
 
 class TFLiteWakewordRecognizerTest: XCTestCase {
     
-    let tflwr = TFLiteWakewordRecognizer.sharedInstance
     let context = SpeechContext()
     let delegate = TFLiteWakewordRecognizerTestDelegate()
     let config = SpeechConfiguration()
-    
+    var tflwr: TFLiteWakewordRecognizer?
```

Comment:
Why bother making it optional in tests?

Reply:
because the initializer requires some state that `setUp` provides